src/Faker/Provider/id_ID/Color.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
Please remove this phpdoc

Reply:
aaah i am sorry, i forget it, will remove it

Reply:
done i have it remove it